src/main/java/com/test/EmployeeTestApplication.java

```diff
@@ -1,51 +0,0 @@
-package com.test;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.List;
-
-import org.springframework.boot.CommandLineRunner;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.DeserializationFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.test.service.OrganisationService;
-import com.test.model.*;
-
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
-
-
-@SpringBootApplication
-@EnableWebMvc
-@EnableSwagger2
-public class EmployeeTestApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(EmployeeTestApplication.class, args);
-	}
-	@Bean
-	   public Docket productApi() {
-	      return new Docket(DocumentationType.SWAGGER_2).select()
-	         .apis(RequestHandlerSelectors.basePackage("com.test")).build();
-	   }
-	/*
-	 * @Bean CommandLineRunner runner(OrganisationService orgService) { return args
```

Comment:
@Rinshakp
What was this code used for and why is it commented?
The commented code has to be removed before committing the changes.